### PR TITLE
Driver Ready Check tweaks 

### PR DESCRIPTION
**Pull request by dandaly**
Tweaks based on WG feedback. Also added comment on how idpf implements this

### CODE REVIEW

File: COORDINATION.md [hunk 1]
```diff
@@ -66,6 +68,21 @@ The virtio-blk device presents its driver in an option ROM (OROM) for UEFI / BIO
 
 The NVMe device driver will poll the CSTS.rdy bit to ensure that infrastructure backend is ready before reading or writing.
 
+### idpf
+
```
Comment (glimchb) [inline COORDINATION.md:72]: please consider adding to terms table what IDPF is... I guess `Infrastructure Data-Plane Function` ?

File: COORDINATION.md [hunk 2]
```diff
@@ -52,7 +52,9 @@ Infrastructure devices acting as either a Host peripheral or as an independent e
 
 ### Assumptions
 
-The infrastructure devices must be ready on the PCIe bus(es) at enumeration time which allows the driver to address the race condition.
+The infrastructure devices must follow the PCI spec and be ready on the bus(es)
+at enumeration time, and/or hot plug devices in and out dynamically.
```
Comment (glimchb) [inline COORDINATION.md:56]: why two lines ? by mistake or intentional ?

Comment (glimchb) [inline COORDINATION.md:56]: I see markdown checker linter failed on 2 blank lines here... line 58

**Review by glimchb — CHANGES_REQUESTED** [review]
linter fails, all the rest of the comments are optional

File: COORDINATION.md [hunk 3]
```diff
@@ -66,6 +68,21 @@ The virtio-blk device presents its driver in an option ROM (OROM) for UEFI / BIO
...
+
+The idpf device will come up with the link down, and will notify the driver
+that it is ready by bringing the link up.  The driver will wait to send on
+the device until the link is brought up.
+
+### Summary
+
+- **Pros:** No assumptions or changes required on the host, including BIOS.
+Infrastructure devices can be used to provision the host they
+are running inside. Devices presented can be standard and/or software defined.
```
[the remark this answers is not in the document]
Comment (dandaly) [reply COORDINATION.md:81]: This feels like a detail in a specific implementation, not something fundamental to DPU or IPU. There certainly exists software defined functions in deployment today that can meet the PCI spec.  I think we should take the PCI spec as a boundary condition, and see if we can make DPUs and IPUs work in that first.

File: COORDINATION.md (hunk 2 above)
Comment (dandaly) [reply COORDINATION.md:56]: Yes I added a line break so it wouldn't wrap beyond 80 characters.

File: COORDINATION.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (glimchb) [reply COORDINATION.md:80]: Yep. Just saying maybe we can’t meet PCI spec…

**Review by glimchb — APPROVED** [review]
Thanks for the fixes

File: COORDINATION.md (hunk 1 above)
Comment (glimchb) [reply COORDINATION.md:71]: added https://github.com/opiproject/opi-prov-life/commit/3846f52fb8fd8e779929541b8e3350bb160165e3

**Pull request merged**